docs/content/integrations/airbyte.mdx

```diff
@@ -241,6 +245,60 @@ defs = Definitions(
 )
 ```
 
+</TabItem>
+
+<TabItem name="Without an I/O manager">
+
+In this case, we have an Airbyte connection which is storing data in the `stargazers` table in our Snowflake warehouse. Since we are not using an I/O manager to fetch the data in downstream assets, we will use `deps` to define dependencies. Then within the downstream asset, we can fetch the data if necessary or launch other commands that work with data in external processes.
```

Comment:
```suggestion
In this case, we have an Airbyte connection that stores data in the `stargazers` table in our Snowflake warehouse. Since we are not using an I/O manager to fetch the data in downstream assets, we will use `deps` to define dependencies. Then within the downstream asset, we can fetch the data if necessary or launch other commands that work with data in external processes.
```